### PR TITLE
Add KibelaReader

### PR DESCRIPTION
**Pull request by higebu**
This PR adds the KibelaReader. [Kibela](https://kibe.la/) is a Japanese knowledge sharing tool.

### CODE REVIEW

**Comment by jerryjliu** [comment]
@higebu thanks! will add soon

**Review by jerryjliu — CHANGES_REQUESTED** [review]
@higebu main ask is could you add an entry to library.json?

**Comment by higebu** [comment]
@jerryjliu Thanks! I'll add it soon.

**Comment by higebu** [comment]
@jerryjliu I added the entry for kibela to library.json.

**Review by jerryjliu — APPROVED** [review]

**Comment by higebu** [comment]
@jerryjliu Sorry. I fixed the syntax error.
https://github.com/emptycrown/llama-hub/actions/runs/5328581150/jobs/9656127995#step:6:73

File: llama_hub/kibela/base.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+"""LLama Kibela Reader"""
+from typing import Dict, List, Optional, TypeVar, Generic
+from gql import Client, gql
```
Comment (jerryjliu) [inline llama_hub/kibela/base.py:3]: sorry i think you might have to do a lazy import here for tests to pass

Comment (higebu) [reply llama_hub/kibela/base.py]: Thank you. I fixed it.

**Pull request merged**